src/main/java/uk/gov/ons/ctp/response/casesvc/domain/repository/CaseIacAuditRepository.java

```diff
@@ -2,8 +2,22 @@
 
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
 import uk.gov.ons.ctp.response.casesvc.domain.model.CaseIacAudit;
 
-/** Jpa Repository for CaseIacAudit table */
+/** JPA Data Repository for CaseIacAudit table. */
 @Repository
-public interface CaseIacAuditRepository extends JpaRepository<CaseIacAudit, Integer> {}
+@Transactional(readOnly = true)
+public interface CaseIacAuditRepository extends JpaRepository<CaseIacAudit, Integer> {
+
+  /**
+   * Find the latest case iac
+   *
+   * @param caseFK
```

Comment:
It's worth deleting the `@param` and `@return` if there is no more context to add

Reply:
fixed